java-sdk-springboot-example/src/main/resources/app-keypair.pem

```diff
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEowIBAAKCAQEAtoOE2ZuTmsRIsAqcyG7nKb2D26WqMP8+iNAT0BydWoMuWEOj
+MIIEogIBAAKCAQEAtoOE2ZuTmsRIsAqcyG7nKb2D26WqMP8+iNAT0BydWoMuWEOj
```

Comment:
why has this changed?

Reply:
It contained the dashboard encoding bug which means that in V1.1 of the SDK this key file can't be read. This is the result of re-encoding the PEM with the trick suggested by Henry with OpenSSL.

Reply:
Although, it's worth noting that either way - there is no way someone who checked out this example could run the app with this file (either before or after this change). This is because the user would not know the SDK ID or the dashboard application ID to allow them to complete a login flow against an app that is configured to redirect them to their localhost. I think the file serves more as a placeholder to show the user where to place their own PEM file once they download it from dashboard.